### DISCUSSION TITLE
Ma page ne s'affiche pas sur GitHub Pages

### DISCUSSION THREAD

**Discussion by ClementH9** [community/community#102770] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Bonjour, j'essaie d'afficher ma page sur GitHub pages, mais avec le chemin suivant : https://clementh9.github.io/HotTakes/front/src/index.html j'obtiens une page blanche
Comment faire pour pouvoir y avoir accès svp ? Je ne vois pas l'erreur. Merci

**Comment by queenofcorgis** [comment]
Hi @ClementH9 , thanks for participating!
We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be closing and locking any posts in languages other than English for now, including this one.
